build.sh

```diff
@@ -0,0 +1,8 @@
+# This is for lazy people -> just type `./build.sh` in the terminal
+mkdir -p build
+cd build
+cmake ..
+make
+src/numsim ../settings.txt
+paraview out/output_*  # or manually open in paraview
+# You may need to change access with `chmod +x build.sh`
```

Comment:
By running `make install` instead of `make`, the `numsim` executable will be copied to the `build` folder, as mentioned in the docs (although the docs use `make`. I think it was a typo)